openpectus/lang/exec/uod.py

```diff
@@ -901,7 +901,7 @@ def get_additive_options(self) -> list[str]:
         if "option" not in self.get_named_groups():
             return []
         start = self.regex.index("|(") + len("|(")
-        end = self.regex.index("|\+)+))\s*")
+        end = self.regex.index("|\\+)+))\\s*")
```

Comment:
@nneskildsf Got two 'Unsupported escape sequence' warnings that we missed earlier. Removing the backslashes breaks the unit test. Adding backslashes fixes the warnings while still passing the unit test. So we're good, right?